web/test/e2e/docunit.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from "@playwright/test"
+
+test.describe("generate and delete doc units", () => {
+  let docUnitId: string
+
+  test("generate doc unit", async ({ page }) => {
+    await page.goto("/")
+
+    const newDocUnitButton = await page.waitForSelector(
+      "button >> text=Neue Dokumentationseinheit"
+    )
+    newDocUnitButton.click()
+
+    await page.waitForSelector("text=Rechtsprechung")
+    docUnitId = await page.locator(".panel-id").innerText()
```

Comment:
I think, we should try to avoid using css selectors for tests. Since they rely on implementation details, they tend to be brittle. [Testing-Library](https://testing-library.com/docs/queries/bylabeltext) offers some neat selectors (I'm about to commit the installation).

Reply:
I'm very interested in this because I know many QA worker, which has this problem with the change of selectors by the developer.